requirements.txt

```diff
@@ -1 +1,4 @@
--r requirements/default.txt
+# requirements are in setup.cfg as extras
+# https://caremad.io/posts/2013/07/setup-vs-requirement/
+
+-e ".[dev]"
```

Comment:
Unclear whether this should be `dev` or `all`... Can you comment on why you chose dev?

Reply:
It might also be worth leaving the other `requirements/*.txt` files with the appropriate `-e "..[]"`, *if that works*.

Reply:
> Unclear whether this should be `dev` or `all`... Can you comment on why you chose dev?

I was kinda reserving `[all]` for the usage that @sofroniewn mentioned in https://github.com/napari/napari/pull/1246#issuecomment-636509692, which is more of a user-facing extra, that might be a place where we put all of the "batteries included" plugins (he mentioned `napari-svg` and stuff) ... whereas `[dev]` was meant to be the "kitchen sink" of tools for napari contributors.

Reply:
> It might also be worth leaving the other requirements/*.txt files with the appropriate -e "..[]", if that works.

I can definitely add back do this if you wanted it... I think it would work.   my (slight) preference would to be leave it out.  seems unnecessarily repetitive to me (though i get that those used to `pip install -r` would look there first)

Reply:
Yeah I would add the files back, so existing workflows aren't screwed up. It's not so much about future users as people who are already developing with napari.